.github/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+## What?
+
+Here goes your feature/bug-fix proposal
+
+### Type of change?
+
+- [ ] [build]: Changes that affect the build system or external dependencies (example scopes: gulp, broccoli, npm)
+- [ ] [ci]: Changes to our CI configuration files and scripts (example scopes: Travis, Circle, BrowserStack, SauceLabs)
+- [ ] [docs]: Documentation only changes
+- [ ] [feat]: A new feature
+- [ ] [fix]: A bug fix
+- [ ] [perf]: A code change that improves performance
+- [ ] [refactor]: A code change that neither fixes a bug nor adds a feature
+- [ ] [style]: Changes that do not affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
+- [ ] [test]: Adding missing tests or correcting existing tests
+
+Note: Remove the non-checked options
+
+## Why?
+
+Here you express the reason it is necessary (keep the _e daí?_ factor in mind)
+
+## How?
+
+Here you give a little more detail on the implementation or justify any unorthodox choice or practice.
+
+## How has this been tested?
+
+- [ ] Unit tests
+- [ ] Integration tests
+- [ ] Benchmark tests
+
+
+# Checklist:
```

Comment:
This looks like a development checklist and not a PR one. maybe change the title to something like `before submitting a PR please make sure to:`. what do you think?

Reply:
Great tip.
I'll do this.